beacon-chain/sync/validate_aggregate_proof.go

```diff
@@ -59,6 +59,13 @@ func (r *Service) validateAggregateAndProof(ctx context.Context, pid peer.ID, ms
 	if seen {
 		return false
 	}
+	r.pendingAttsLock.Lock()
+	if !r.validateBlockInAttestation(ctx, m) {
```

Comment:
maybe rename to. it is not clear
```suggestion
	if !r.attestationBlockInDB(ctx, m) {
```